data/human/escorts.txt

```diff
@@ -0,0 +1,19 @@
+#Example of an escort definition:
+#
+#mission "Escort"
+#	name "Escort"
+#	description "This Captain will Escort you for 200 credits per day."
+#	repeat
+#	job
+#	"escort salary" 200
+#	source
+#		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+#	to complete
+#		never
+#	npc accompany save
+#		government "Merchant"
+#		personality escort heroic
+#		fleet
+#			names "civilian"
+#			variant
+#				"Sparrow"
```

Comment:
Could you move all of this into the PR's description?
That's where people will look how to use the new feature. If you're worried it'll take too much space, use this:
```
<details><summary>Usage example.</summary>

Here, after the blank line, put the example.
</details>```